src/test/java/com/karankumar/bookproject/backend/goal/CalculateReadingGoalTest.java

```diff
@@ -100,4 +100,12 @@ void testHowFarAheadOrBehindSchedule(){
         assertEquals(45, howFarAheadOrBehindSchedule(113,41));
         assertThat(howFarAheadOrBehindSchedule(113,86)).isZero();
     }
+
+    //Test to test the change in the method booksToReadFromStartOfYear
+    @Test
+    void testBooksToReadFromStartOfYear(){
+        int booksToReadThisYear = 104;
+        int booksToReadFromStartOfYear = CalculateReadingGoal.booksToReadFromStartOfYear(booksToReadThisYear);
```

Comment:
Could you bring the initialisation part down one line so that we can adhere to the 100 character column limit?